lib/shared/log/tasks.js

```diff
@@ -29,22 +32,23 @@ function logTasks(tree, opts, getTask) {
 
 function printTaskTree(tree, opts) {
   var lines = [];
-  lines.push({ label: tree.label });
   var maxLabelWidth = 0;
 
   tree.nodes.forEach(function(node, idx, arr) {
     var isLast = idx === arr.length - 1;
     var w = createTreeLines(node, lines, opts, 1, '', isLast);
-    maxLabelWidth = Math.max(maxLabelWidth, w || 0);
+    maxLabelWidth = Math.max(maxLabelWidth, w || /* istanbul ignore next */ 0);
```

Comment:
Why add this comment now?

Reply:
For increasing coverage. However, in the first place,`|| 0` is unnecessary.